yarn.lock

```diff
@@ -2420,6 +2420,28 @@
     lodash "^4.17.21"
     node-fetch "^2.6.1"
 
+"@dendronhq/common-all@^1.00.0":
```

Comment:
how was dendronhq added to dependencies? if you add using `lerna bootstrap`, this should not be here. can you reset the yarn.lock and try running lerna bootstrap instead?